src/scripts/main.js

```diff
@@ -1 +1,106 @@
 'use strict';
+
+const bodyDoc = document.querySelector('body');
+let rightClick;
+let leftClick;
+
+function makeDiv(className) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.setAttribute('class', className);
+  bodyDoc.appendChild(div);
+
+  return div;
+}
+
+const firstPromise = () => {
+  return new Promise((resolve, reject) => {
+    document.addEventListener(
+      'click',
+      () => {
+        leftClick = true;
+
+        resolve('First promise was resolved');
+      },
+      { once: true },
+    );
+
+    // eslint-disable-next-line prefer-promise-reject-errors
+    setTimeout(() => reject('First promise was rejected!'), 3000);
+  });
+};
+
+const secondPromise = () => {
+  return new Promise((resolve) => {
+    document.addEventListener(
+      'click',
+      () => {
+        leftClick = true;
+
+        resolve('Second promise was resolved');
+      },
+      { once: true },
+    );
+
+    document.addEventListener(
+      'contextmenu',
+      () => {
+        rightClick = true;
```

Comment:
The variable `rightClick` is being set but never used elsewhere. If this is intended for future use, consider adding a comment to explain its purpose.